book.toml

```diff
@@ -15,11 +15,15 @@ renderer = ["html"]
 
 [preprocessor.admonish]
 command = "mdbook-admonish"
-assets_version = "2.0.2" # do not edit: managed by `mdbook-admonish install`
```

Comment:
Could you keep unrelated config changes out of this PR? This makes CI fail at the moment.

Reply:
no problem. i just assumed it was okay to commit together because of all the version errors i got when i first started building from source.